components/PrivateHeader.vue

```diff
@@ -13,48 +13,27 @@ function onChange(value: string) {
 
 <template>
   <div class="header">
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
-      <div class="collapse navbar-collapse">
-        <ul class="navbar-nav mr-auto">
-          <li class="nav-item">
-            <NuxtLink v-if="authStore.token" :to="localePath('/user')" class="dogerBlue nav-link">
-              {{ $t("links.users") }}
-            </NuxtLink>
-          </li>
-          <li class="nav-item">
-            <NuxtLink v-if="authStore.token" :to="localePath('/about')" class="dogerBlue nav-link">
-              {{ $t("links.about") }}
-            </NuxtLink>
-          </li>
-          <li class="nav-item">
-            <NuxtLink v-if="authStore.token" :to="localePath('/profile')" class="dogerBlue nav-link">{{
-              $t("links.profile")
-            }}</NuxtLink>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#" @click="logout">Logout</a>
-          </li>
-          <li class="nav-item">
-            <NuxtLink :to="localePath('/rendering')" class="color-dogerBlue nav-link">Rendering</NuxtLink>
-          </li>
-          <li class="nav-item dropdown">
-            <select v-model="selectedValue" class="nav-link dropdown-toggle" @change="onChange(selectedValue)">
-              <option v-for="item in locales" :key="item.code" :value="item.code">
-                {{ item.name }}
-              </option>
-            </select>
-          </li>
-        </ul>
-      </div>
+    <nav>
```

Comment:
did you put and design again?? azmul bhai changed the project to bootstrap.

Reply:
thanks for noticing. due to the linter issue my files and changes were being removed so undoing caused this. Fixing it.